lib/geoengineer/resources/aws/iam/aws_iam_role.rb

```diff
@@ -37,7 +37,8 @@ def self._fetch_remote_resources(provider)
       r.merge({ name: r[:role_name],
                 _geo_id: r[:role_name],
                 _terraform_id: r[:role_name],
-                assume_role_policy: URI.decode(r[:assume_role_policy_document]) })
+                assume_role_policy: URI.decode(r[:assume_role_policy_document]),
+                max_session_duration: r[:max_session_duration] })
```

Comment:
Why do you need to manually set this? If you're just setting this to `r[:max_session_duration]`, shouldn't `r[:max_session_duration]` already be set on `r`? (notice the `merge` above).

Reply:
Oh wow you're right. I guess the resource already supports `max_session_duration` 🤷‍♂️ 

Closing this PR. Thanks for the comment!